extensions/opentelemetry/runtime/pom.xml

```diff
@@ -55,6 +59,12 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-grpc</artifactId>
             <optional>true</optional>
+            <exclusions>
+                <exclusion>
+                    <groupId>io.quarkus</groupId>
+                    <artifactId>quarkus-security-runtime-spi</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
This exclusion is not required; the explicit dep further up will override this transitive dep (e.g. see dependency:tree).

Reply:
yes, but if someone removes the `quarkus-security-runtime-spi` dependency the build will still pass, this way it will fail. It's a safeguard.